### DISCUSSION TITLE
JavaFx 24 minimum requirement JDK version is 22, so should we bump the version to 22.x?

### DISCUSSION THREAD

**Discussion by chengenzhao** [AlmasB/FXGL#1391] [General]
Hi:

[The minimum JDK requirement for JavaFX 24 is 22](https://github.com/openjdk/jfx/pull/1588)
& 21.x is not applicable for JavaFX 24
so is that mean we should bump the FXGL future version e.g. 21.2 to 22.x?

**Comment by AlmasB** [comment]
Hia, currently FXGL bumps the JDK version with LTS releases. I believe the next LTS release will be Java 25, in Sept 2025, so we might upgrade all in one go then.

I've been a bit out of the loop on FX releases. Is there anything in particular JavaFX 24 brings? If nothing major, I think we can wait for Java / JavaFX 25. Otherwise, we could go for an intermediate FXGL 24 release using JDK 24.

**Reply by AlmasB** [reply]
@chengenzhao 
I've updated the dev snapshot to 25 (tentative releases in Sept 2025) and bumped base JDK to 23

**Reply by chengenzhao** [reply]
It looks great
